src/main/java/frc/robot/Constants.java

```diff
@@ -4,12 +4,50 @@
 
 package frc.robot;
 
-/**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
- * constants. This class should not be used for any other purpose. All constants should be declared
- * globally (i.e. public static). Do not put anything functional in this class.
- *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
- * constants are needed, to reduce verbosity.
- */
-public final class Constants {}
+public final class Constants {
+
+    public static final class DriveTrain {
+
+        public static final class TalonR {
+            public static final int TALONR_ID = 1;
```

Comment:
Is there any other possible ID?
```suggestion
            public static final int ID = 1;
```
this would make the usage cleaner:
```java
constants.DriveTrain.TalonR.ID
//instead of
constants.DriveTrain.TalonR.TALONR_ID
```
which is a bit redundant